types/connector_lock.yaml

```diff
@@ -0,0 +1,37 @@
+description: EV and EVSE board support types
+types:
+  ConnectorLockError:
+    description: >-
+      Errors for Connector Lock
+
+      These following error types give more details about connector lock errors.
+      You should consider setting a MREC_1_ConnectorLockFailure to indicated a generic lock failure with an MREC error code as well
+      as one of the more detailed but custom error codes to specify the reason:
+
+      ConnectorLockCapNotCharged: The capacitor for connector lock motor failed to charge within expected time
+      ConnectorLockUnexpectedOpen: The connector lock feedback does return open while it should be close
+      ConnectorLockUnexpectedClose: The connector lock feedback does return closed while it should be open
+      ConnectorLockFailedLock: The connector lock failed to lock (feedback still returns open)
+      ConnectorLockFailedUnlock: The connector lock failed to unlock (feedback still returns closed)
```

Comment:
What is the difference to the `ConnectorLockUnexpectedOpen`?

Reply:
their are two different errors that can happen: If you intent to lock/unlock it it may not work. The second error is: if it was locked for quite a while and there is no intent to open it but it still does.

Reply:
Ok, understood. I'm not sure if this is a nice interface since the `ConnectedLockFailedLock`(and unlock) are results from an rpc call. Maybe it would be better to communicate them as part of the rpc call?

Reply:
refactored with the new error framework, so this type is gone